src/Magnum/DebugTools/ColorMap.h

```diff
@@ -204,6 +204,20 @@ MAGNUM_DEBUGTOOLS_EXPORT Containers::StaticArrayView<256, const Vector3ub> infer
 */
 MAGNUM_DEBUGTOOLS_EXPORT Containers::StaticArrayView<256, const Vector3ub> viridis();
 
+/**
+@brief Cool-Warm colormap
+@m_since{2020,06}
+
+@image html colormap-cool_warm.png width=100%
+
+@m_class{m-block m-primary}
+
+@thirdparty Created by [Kenneth Moreland](https://www.kennethmoreland.com/color-advice/).
+    It is a diverging (double-ended) color map with a smooth transition in the middle to prevent artifacts at the midpoint.
+    Although not isoluminant, this color map avoids dark colors to allow shading cues throughout.
```

Comment:
Hmm, I wanted to point out that I don't like how it doesn't have uniform brightness, but this comment mentions that, so all good :)

I tried to find some licensing info, but didn't find any, so I guess I'll assume Public Domain, then.